tracee-rules/engine/engine.go

```diff
@@ -334,3 +334,12 @@ func analyzeEventOrigin(event tracee.Event) string {
 		return EVENT_HOST_ORIGIN
 	}
 }
+
+// GetSelectedEvents returns the event selectors that are relevant to the currently loaded signatures
+func (engine *Engine) GetSelectedEvents() []types.SignatureEventSelector {
```

Comment:
Don't we want the signature for this to be:
`GetSelectedEvents() []string`
As described in #936 ?

Reply:
I think that we'll need the entire event selector because potentially it would include more filtering data for tracee (fixed the example in the issue)

Reply:
Ok. Anyways, I think duplications should be removed in the returned value

Reply:
they are removed (see the test)